can/interfaces/vector/canlib.py

```diff
@@ -0,0 +1,157 @@
+# -*- coding: utf-8 -*-
+"""
+Ctypes wrapper module for Vector CAN Interface on win32/win64 systems
+Authors: Julien Grave <grave.jul@gmail.com>
+"""
+# Import Standard Python Modules
+# ==============================
+import ctypes
+import logging
+import re
+import sys
+
+# Import Modules
+# ==============
+from can import BusABC, Message
+
+# Define Module Logger
+# ====================
+LOG = logging.getLogger('can.vector')
+
+# Import safely Vector API module for Travis tests
+if sys.platform == 'win32':
+    import vxlapi
+else:
+    LOG.warning('Vector API does not work on %s platform', sys.platform)
+    vxlapi = None
+
+# Define Module Constants
+# =======================
+RX_MSG_RE = re.compile(
+    r'^RX_MSG c=(?P<channel>\d+), t=(?P<timestamp>\d+), (id=(?P<id>[\d\w]+) l=(?P<dlc>\d+)|type=(?P<type>\d+)),\s+(?P<data>[\d\w_]+)(?: (?P<flag>[\d\w]+))? tid=(?P<tid>\d+)$'
+).match
+CHIP_STATE_RE = re.compile(
+    r'^CHIP_STATE c=(?P<channel>\d+), t=(?P<timestamp>\d+), busStatus=\s?(?P<busStatus>\w+), txErrCnt=(?P<txErrCnt>\d+), rxErrCnt=(?P<rxErrCnt>\d+)$'
+).match
+
+
+class VectorBus(BusABC):
+    """The CAN Bus implemented for the Vector interface."""
+
+    def __init__(self, channel, can_filters=None, **config):
```

Comment:
It would be nice to support ``can_filters`` also since there is the ``xlCanSetChannelAcceptance()`` function. Since the HW only supports one filter you could ignore it if more than one is given.